Project/src/MakeCall/Lobby.js

```diff
@@ -1,24 +1,34 @@
-import React, { useEffect, useState } from "react";
+import React from "react";
 import { PrimaryButton } from 'office-ui-fabric-react';
+import { Features } from '@azure/communication-calling';
 
 // Lobby react function component
-const Lobby = ({ call }) => {
-    const [lobby, setLobby] = useState(call.lobby);
-    const [lobbyParticipantsCount, setLobbyParticipantsCount] = useState(lobby.participants.length);
+export default class Lobby extends React.Component {
+    constructor(props) {
+        super(props);
+        this.call = props.call;
+        this.lobby = this.call.lobby;
+        
+        this.capabilitiesFeature = this.call.feature(Features.Capabilities);
```

Comment:
We are already initializing capabilitiesFeature on CallCard, can we leverage it?

Reply:
The reason I initialize it in both Lobby and RemoteParticipantCard is because it needs to get the capabilitiesChangedEvent (same with RaiseHandFeature and SpotlightFeature), so that the buttons will only show up if the user has access to manage Lobby